projects/igniteui-angular/src/lib/core/grid-selection.ts

```diff
@@ -175,7 +175,7 @@ export class IgxGridCRUDService {
                 return;
             }
 
-            if (this.row && !this.sameRow(this.cell.id.rowID)) {
+            if (this.row && !this.sameRow(this.cell.id.rowIndex)) {
```

Comment:
would prefer if this used the top level prop of `IgxCell` - `this.cell.rowIndex` just so it's not tied to index being part of the cell id.